packages/lesmis/src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { Link as RouterLink } from 'react-router-dom';
+import MuiBreadcrumbs from '@material-ui/core/Breadcrumbs';
+import MuiLink from '@material-ui/core/Link';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import { useCurrentOrgUnitData } from '../api';
+
+const StyledBreadcrumbs = styled(MuiBreadcrumbs)`
+  font-size: 0.75rem;
+  line-height: 0.875rem;
+  text-transform: capitalize;
+
+  .MuiBreadcrumbs-separator {
+    margin-left: 0.2rem;
+    margin-right: 0.3rem;
+
+    svg {
+      font-size: 0.75rem;
+    }
+  }
+`;
+
+const ActiveSegment = styled.span`
+  color: ${props => props.theme.palette.primary.main};
+`;
+
+const Link = props => <MuiLink color="inherit" {...props} component={RouterLink} />;
+
+const getSegments = (orgUnits, orgUnitCode, heirarchy = []) => {
+  const orgUnit = orgUnits.find(entity => entity.organisationUnitCode === orgUnitCode);
+
+  if (!orgUnit) {
+    return [];
+  }
+
+  const newHeirarchy = [...heirarchy, orgUnit];
```

Comment:
```suggestion
  const newHierarchy = [...hierarchy, orgUnit];
```
Better do a find/replace on this

Reply:
Also
```suggestion
  const newHeirarchy = [orgUnit, ...heirarchy];
```
Would avoid the `reverse` (no biggy either way - whatever you find easier to think about)

Reply:
💯